static/gsApp/views/subscriptionPage/headerCards/usageCard.tsx

```diff
@@ -39,7 +40,11 @@ export function UsageCard({subscription, organization}: UsageCardProps) {
     ondemand: theme.chart.getColorPalette(5)[1],
   } as const;
 
-  if (!intervalPrice || !shouldSeeSpendVisibility(subscription)) {
+  if (
+    !intervalPrice ||
+    !shouldSeeSpendVisibility(subscription) ||
+    hasCheckoutV3(organization) // TODO(checkout v3): remove this, this is temporary until the real header cards for V3 are implemented
```

Comment:
this is just to make it look like it's not broken temporarily for anyone who accesses the internal orgs and sees the placeholder i put to make knip happy